File: src/mosaic/ua/ksn/fmg/event/click/LeftDownResult.java
package ua.ksn.fmg.event.click;

import java.util.ArrayList;
import java.util.List;

import ua.ksn.fmg.model.mosaics.cell.BaseCell;

public class LeftDownResult {
	public List<BaseCell> needRepaint;

	public LeftDownResult()
	{
		needRepaint = new ArrayList<BaseCell>();
	}
}

File: src/mosaic/ua/ksn/fmg/event/click/LeftDownResult.cs
using System.Collections.Generic;
using ua.ksn.fmg.model.mosaics.cell;

namespace ua.ksn.fmg.Event.click {

public class LeftDownResult {
	public IList<BaseCell> needRepaint;

	public LeftDownResult()
	{
		needRepaint = new List<BaseCell>();
	}
}
}